spec/queryAction.ts

```diff
@@ -0,0 +1,9 @@
+import * as faker from 'faker';
+import { QueryAction } from '../src/api/dataops/queries/baseQuery';
+
+export const getRandomQueryAction = () => faker.helpers.randomize([
+  QueryAction.delete,
+  QueryAction.insert,
+  QueryAction.select,
+  QueryAction.update,
```

Comment:
I would change this to `faker.helpers.randomize(Object.values(QueryAction));`

Reply:
done!